### ISSUE TITLE
Delete old patchdemo server

### ISSUE THREAD

**Issue by edg2s** [MatmaRex/patchdemo#257]
I don't think we need it for anything?

**Comment by MatmaRex** [comment]
No, I left it around in case we forgot about something important.

I was going to keep it until the Cloud people send their annual nag email about unused instances, but you can delete it if you want.
